package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "blue-blocker",
-	"version": "0.3.5",
+	"version": "0.3.7",
```

Comment:
uh so we're just going to skip 0.3.6?

Reply:
Maybe? I did update the version strings to "0.3.6" in baf074a but forgot to use `node version`. I'm also not super sure about how to tag the changes, is the whole pull request a patch or would each part of it be it's own patch?

Reply:
I think the whole PR would be a patch (since we usually squash when merging), but let's double-check with @kheina just to be sure...

Reply:
I'll update the version strings then :)